docs/getting-started.md

```diff
@@ -97,41 +109,54 @@ You must have Windows Subsystem for Linux installed. To install WSL, refer to Mi
       ```bash
       python3 -m pip install semgrep
       ```
-
   2. Confirm installation:
       ```sh
       semgrep --version
       ```
-
-  3. Run recommended Semgrep Registry rules:
-  3. Log in to Semgrep Cloud Platform and run a scan using recommended rules:
+  3. Sign up or log in to Semgrep Cloud Platform (SCP):
+      ```sh
+      semgrep login
+      ```
+      This command automatically opens a new tab or browser window. You can also click the link in the CLI to complete the step.
+  4. A modal appears after logging in to SCP. Click **Create an organization**. Note that you can further integrate organizations (orgs) with GitLab accounts and GitHub accounts, including personal and org accounts, after you complete this procedure.
+  5. After creating an organization in the previous step, return to the CLI and run a scan:
       ```sh
-      semgrep login && semgrep ci
+     semgrep ci
       ```
 
 </TabItem>
 
 <TabItem value='Docker'>
 
-  1. Pull latest image locally:
+  1. Pull the latest image locally:
      ```sh
      docker pull returntocorp/semgrep
      ```
-   
   2. Confirm version:
       ```sh
       docker run --rm returntocorp/semgrep semgrep --version
       ```
-
-  3. Run recommended Semgrep Registry rules:
-      1. On **macOS or Linux**, in the directory to scan:
-        ```bash
-        docker run --rm -v "${PWD}:/src" returntocorp/semgrep semgrep login && semgrep ci
-        ```
-        The provided `-v` option mounts the current directory into the container to be scanned. Change directories locally or provide a specific local directory in the command to scan a different directory.
-      2. On **Windows**, in the directory to scan:
+  3. For **macOS or Linux** Docker users, perform the following steps:
+     1. Sign up or log in to Semgrep Cloud Platform (SCP):
+         ```sh
+        docker run --rm -v "${PWD}:/src" returntocorp/semgrep semgrep login
```

Comment:
This isn't correct - the semgrep login command doesn't require mounting the source directory.
```suggestion
        docker run --rm returntocorp/semgrep semgrep login
```